GalaxyZooTouchTable/GalaxyZooTouchTable/Models/ClassificationPanelViewModelFactory.cs

```diff
@@ -0,0 +1,45 @@
+﻿using GalaxyZooTouchTable.Lib;
+using GalaxyZooTouchTable.Services;
+using GalaxyZooTouchTable.ViewModels;
+using System;
+
+namespace GalaxyZooTouchTable.Models
+{
+    public class ClassificationPanelViewModelFactory : IClassificationPanelViewModelFactory
+    {
+        private IPanoptesRepository _panoptesRepo;
+        private IGraphQLRepository _graphQLRepo;
+
+        public ClassificationPanelViewModelFactory(IPanoptesRepository panoptesRepo, IGraphQLRepository graphQLRepo)
+        {
+            if (panoptesRepo == null || graphQLRepo == null)
+            {
+                throw new ArgumentNullException("RepoDependency");
+            }
+
+            _panoptesRepo = panoptesRepo;
+            _graphQLRepo = graphQLRepo;
+        }
+
+        public ClassificationPanelViewModel Create(UserType type)
```

Comment:
👍 I like this refactor

Reply:
Yes, this was originally what prompted this PR and is necessary to get testing wired up.